Admin/AbstractAdmin.php

```diff
@@ -2172,12 +2172,14 @@ public function getTranslationDomain()
      *
      * @deprecated since 3.9, to be removed with 4.0
      */
-    public function setTranslator(TranslatorInterface $translator)
+    public function setTranslator(TranslatorInterface $translator, $deprecation = true)
```

Comment:
Alright, I'll change it (although I think nobody ever would rewrite this method).